### DISCUSSION TITLE
Welcome to chatnio Discussions!

### DISCUSSION THREAD

**Discussion by zmh-program** [coaidev/coai#256] [Announcements | 公告]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BryceWG** [comment]
Do we need to choose the language for communication?

[a comment by an automated account is not shown]

**Comment by tom-za-wa** [comment]
hi . ..... we can speak #Americano

[a comment by an automated account is not shown]
